tests/coreTest.php

```diff
@@ -94,4 +95,23 @@ function testSiteSelectionViaCwd() {
     }
     chdir($cwd);
   }
+
+  function testOptionsUri() {
+    // Put a yml file in the drush folder.
+    $drush_directory = sprintf('%s/drush', $this->webroot());
+    $drush_config_file = sprintf('%s/drush.yml', $drush_directory);
```

Comment:
whats the benefit of using sprintf here instead of concatenation? Or`Path::join($this->webroot(), 'drush/drush.yml');` Thats the webmozart/path library .. .Actually, if you use the ../drush, you have an equivalent directory that already exists. Thats where sut.alias.yml lives. Its one directory above drupal root.

Reply:
Hm, not sure why my comment was not added here.

I think I wrote something along the lines of this:

There is no real benefit, it is just a matter of taste and coding style. Will update it. :)

Also, great suggestion about the root drush folder. Will use that.